src/middleware/node/middleware.ts

```diff
@@ -93,7 +93,24 @@ export async function middleware(
   }, 9000).unref();
 
   try {
-    const payload = await getPayload(request);
```

Comment:
why are we not doing this in `getPayload`? it's purpose is literally to encapsulate this kind of logic